client/blocks/post-share/publicize-actions-list.jsx

```diff
@@ -75,18 +91,21 @@ class PublicizeActionsList extends PureComponent {
 						{ message }
 					</div>
 				</div>
-				{ this.renderElipsisMenu( actionId ) }
+				{ this.renderElipsisMenu( actionId, message, service ) }
 			</CompactCard>
 		);
 	}
 
-	renderElipsisMenu( publicizeActionId ) {
+	renderElipsisMenu( publicizeActionId, message, service ) {
 		const actions = [];
 		const { translate } = this.props;
 
 		if ( isEnabled( 'publicize-preview' ) ) {
 			actions.push(
-				<PopoverMenuItem key="1" icon="visible">
+				<PopoverMenuItem
+					onClick = { this.togglePreviewModal( message, service ) }
+					key="1"
+					icon="visible">
```

Comment:
Minor suggestion. Close the  PopoverMenuItem putting the `>` in its own line.

```es6
<PopoverMenuItem
	onClick = { this.togglePreviewModal( message, service ) }
	key="1"
	icon="visible"
>
	{ translate( 'Preview' ) }
</PopoverMenuItem>
```
FFTI (My new favorite acronym :-p)